client/src/components/AuthenticationInput.js

```diff
@@ -6,6 +6,7 @@ const AuthenticationInput = styled(Input)`
   background: ${(props) => props.theme.colors.textPrimaryHover};
   color: ${(props) => props.theme.colors.loginFocusText};
   font-size: 22px;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
If you use `inherit` as property the font-family will be inherited from the parent element, so you can save some typing :)